manifests/1.0.0/opensearch-dashboards-1.0.0.yml

```diff
@@ -0,0 +1,12 @@
+---
+schema-version: '1.0'
+build:
+  name: OpenSearch Dashboards
+  version: 1.0.0
+ci:
+  image:
+    name: opensearchstaging/ci-runner:ci-runner-centos7-opensearch-dashboards-build-v2
+components:
+  - name: OpenSearch-Dashboards
+    repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
+    ref: 1.0-healthcheck
```

Comment:
This might be coming from the health check branch OSD cut? https://github.com/opensearch-project/OpenSearch-Dashboards/tree/1.0-healthcheck
And it's using 1.0.0 version at that branch. 
Could you maybe provide more info on this? @kavilla

Reply:
Removed 1.0.0 and 1.x for now

Reply:
> Need 2.3.1 from this PR. Should I edit this brance?

Could you help fix the CI error? Thanks.

Reply:
Our workflow will still create PR regarding of this new branch though we removed it temporarily for now. Need to fix eventually.

Reply:
Yeah it did. Waiting for @kavilla 's response